frontend/src/components/MapView/LeftPanel/AnalysisPanel/ExposureAnalysisActions/index.tsx

```diff
@@ -98,6 +107,12 @@ function ExposureAnalysisActions({
     [analysisCsvData, analysisDefinition],
   );
 
+  const handleToggleReport = useCallback((toggle: boolean) => {
```

Comment:
Is the useCallback hook really necessary here (see https://github.com/WFP-VAM/prism-app/issues/901) ?

Reply:
It is not necessary I will remove it thank you for pointing it out @JulienPathtech